src/core/client/admin/routes/Moderate/Queue/Queue.tsx

```diff
@@ -154,6 +165,20 @@ const Queue: FunctionComponent<Props> = ({
         )}
       />
 
+      {Boolean(
+        viewNewCount &&
+          viewNewCount > 0 &&
+          moderationQueueSort === GQLCOMMENT_SORT.CREATED_AT_ASC
+      ) && (
+        <Flex justifyContent="center" className={styles.viewNewButtonContainer}>
+          <Localized id="moderate-queue-viewNew" $count={viewNewCount}>
+            <Button onClick={onViewNew} className={styles.viewNewButton}>
+              View {viewNewCount} new comments
+            </Button>
+          </Localized>
+        </Flex>
+      )}
+
       {hasMore && (
```

Comment:
Live updates for `oldest` sort should be implemented like on the comment stream. Instead of showing a button here, we should just tell relay that there is more data to fetch und the ui handles the rest.

See: https://github.com/coralproject/talk/blob/f070cbc0e3ac59f4b3cbafc1474c8a3713b59b5e/src/core/client/stream/tabs/Comments/Stream/AllCommentsTab/CommentEnteredSubscription.tsx#L56

Then we can stop the `QueueCommentEnteredSubscription` completely when there is more data. See implementation in stream:
https://github.com/coralproject/talk/blob/f070cbc0e3ac59f4b3cbafc1474c8a3713b59b5e/src/core/client/stream/tabs/Comments/Stream/AllCommentsTab/AllCommentsTabContainer.tsx#L104